jenkins/opensearch/Jenkinsfile

```diff
@@ -134,8 +131,22 @@ pipeline {
             post() {
                 success {
                     node('Jenkins-Agent-al2-x64-c54xlarge-Docker-Host') {
-                        messages = getJenkinsMessages(['build-x64', 'build-arm64'])
-                        publishNotification(":white_check_mark:", "Successful Build", "\n${messages}")
+                        publishNotification(":white_check_mark:", "Successful Build", "\n${getAllJenkinsMessages()}")
+                        script {
+                            def VERSION = sh(script: 'echo ${INPUT_MANIFEST} | grep -Po "[0-9.]+?(?=.yml)"', returnStdout: true).trim()
+                            println VERSION
```

Comment:
Can we delete this line since it's used for our testing.

Reply:
I am ok to leave it there as it shows what version we are retrieving.

Reply:
Change to version + build number for more information overall.